packages/node/examples/express/webhook.js

```diff
@@ -12,35 +11,30 @@ if (!process.env.README_API_KEY) {
 const app = express();
 const port = process.env.PORT || 4000;
 
-app.post('/webhook', express.json({ type: 'application/json' }), (req, res) => {
+// Your ReadMe secret
+const secret = process.env.README_API_KEY;
+
+app.post('/webhook', express.json({ type: 'application/json' }), async (req, res) => {
   // Verify the request is legitimate and came from ReadMe
   const signature = req.headers['readme-signature'];
-  // Your ReadMe secret
-  const secret = process.env.README_API_KEY;
+
   try {
     readme.verifyWebhook(req.body, signature, secret);
   } catch (e) {
     // Handle invalid requests
     return res.status(401).json({ error: e.message });
   }
-  // Fetch the user from the db
-  // eslint-disable-next-line @typescript-eslint/no-use-before-define, @typescript-eslint/no-unused-vars
-  return db.find({ email: req.body.email }).then(user => {
-    return res.json({
-      // OAS Security variables
-      petstore_auth: 'default-key',
-      basic_auth: { user: 'user', pass: 'pass' },
-    });
+
+  // Fetch the user from the database and return their data for use with OpenAPI variables.
+  // const user = await db.find({ email: req.body.email })
```

Comment:
hmm I like the `await` pattern but I do like how the code sample was actually runnable before

Reply:
This one runs too we just aren't assuming what their database system looks like anymore.

Reply:
ohhh my bad I glanced at this line and thought it was passing in the user object:
https://github.com/readmeio/metrics-sdks/blob/2e8c32fb87c678ef4057f3f911dfe1df2ac1d8ee/packages/node/examples/express/webhook.js#L33